src/dnc/azext_dnc/generated/_params.py

```diff
@@ -0,0 +1,120 @@
+        c.argument('location', arg_type=get_location_type(self.cli_ctx), required=False,
+                   validator=get_default_location_from_resource_group)
+        c.argument('tags', tags_type)
+
+    with self.argument_context('dnc controller delete') as c:
+        c.argument('resource_group_name', resource_group_name_type)
+        c.argument('resource_name', type=str, help='The name of the resource. It must be a minimum of 3 characters, '
+                   'and a maximum of 63.', id_part='name')
+
+    with self.argument_context('dnc controller patch') as c:
+        c.argument('resource_group_name', resource_group_name_type)
+        c.argument('resource_name', type=str, help='The name of the resource. It must be a minimum of 3 characters, '
+                   'and a maximum of 63.', id_part='name')
+        c.argument('tags', tags_type)
+
+    with self.argument_context('dnc controller show-detail') as c:
+        c.argument('resource_group_name', resource_group_name_type)
+        c.argument('resource_name', type=str, help='The name of the resource. It must be a minimum of 3 characters, '
+                   'and a maximum of 63.', id_part='name')
+
+    with self.argument_context('dnc delegated-network list') as c:
+        c.argument('resource_group_name', resource_group_name_type)
+
+    with self.argument_context('dnc orchestrator-instance-service list') as c:
+        c.argument('resource_group_name', resource_group_name_type)
+
+    with self.argument_context('dnc orchestrator-instance-service create') as c:
+        c.argument('resource_group_name', resource_group_name_type)
+        c.argument('resource_name', type=str, help='The name of the resource. It must be a minimum of 3 characters, '
+                   'and a maximum of 63.')
+        c.argument('location', arg_type=get_location_type(self.cli_ctx), required=False,
+                   validator=get_default_location_from_resource_group)
+        c.argument('tags', tags_type)
+        c.argument('type_', options_list=['--type'], arg_type=get_enum_type(['SystemAssigned', 'None']), help='The '
+                   'type of identity used for orchestrator cluster. Type \'SystemAssigned\' will use an implicitly '
+                   'created identity orchestrator clusters', arg_group='Identity')
+        c.argument('orchestrator_app_id', type=str, help='AAD ID used with apiserver')
+        c.argument('orchestrator_tenant_id', type=str, help='TenantID of server App ID')
+        c.argument('cluster_root_ca', type=str, help='RootCA certificate of kubernetes cluster base64 encoded')
+        c.argument('api_server_endpoint', type=str, help='K8s APIServer url. Either one of apiServerEndpoint or '
+                   'privateLinkResourceId can be specified')
+        c.argument('private_link_resource_id', type=str, help='private link arm resource id. Either one of '
+                   'apiServerEndpoint or privateLinkResourceId can be specified')
```

Comment:
This parameter is too long to pass linter rule. Please add an alias in options_list.
```suggestion
        c.argument('private_link_resource_id', type=str, options_list=['--private-link-resource-id', '--private-link-id'], help='private link arm resource id. Either one of '
                   'apiServerEndpoint or privateLinkResourceId can be specified')
```

Reply:
Aliased this parameter